linalg/hypre.hpp

```diff
@@ -1753,7 +1753,11 @@ class HypreAMS : public HypreSolver
    /// Construct the gradient and interpolation matrices associated with
    /// @a edge_fespace, and add them to the solver.
    void MakeGradientAndInterpolation(ParFiniteElementSpace *edge_fespace,
-                                     int cycle_type);
+                                     bool trace_space, int p, int cycle_type);
```

Comment:
It would be helpful to document these arguments.

Reply:
Done in [2ab8692](https://github.com/mfem/mfem/pull/3774/commits/2ab8692165118e7e17e3db6a3d47eaa937cc87c3).